backup-mongodb.go

```diff
@@ -82,81 +82,87 @@ func addtoArchive(path string, fileInfo os.FileInfo, err error) error {
 
 	file, err := os.Open(path)
 	if err != nil {
-		fmt.Println("Cannot open file to add to archive: " + path + ", error: " + err.Error())
-		return err
+		log.Panic("Cannot open file to add to archive: "+path+", error: "+err.Error(), err)
```

Comment:
Yes, but I chose Panic() because I want all the deferred functions to execute, especially the one that unblocks the database. Fatal() does os.Exit(1), in which case deferred functions aren't executed before exiting.